app/controllers/user_journey_controller.rb

```diff
@@ -87,4 +96,8 @@ def confirm
   def find_certificate
     @certificate = Certificate.find_by_id(params[:certificate_id])
   end
+
+  def dual_running
+    @dual_running = params[:dual_running]
```

Comment:
if we do something like
```suggestion
    @dual_running = params[:dual_running] == 'no' ? true : nil
```
We don't have to use repeat `if @dual_running == 'no'` in all the views and just use `if @dual_running`